src/walletSdk/horizon/Account.ts

```diff
@@ -0,0 +1,52 @@
+import { Keypair, Transaction, FeeBumpTransaction } from "stellar-sdk";
+
+class AccountKeypair {
+  keypair: Keypair;
+  constructor(keypair: Keypair) {
+    this.keypair = keypair;
+  }
+
+  get publicKey(): string {
```

Comment:
@Ifropc  is there a reason to include both `publicKey` and `accountId` from the keypair? https://github.com/stellar/kotlin-wallet-sdk/blob/e6c801e24e7c716b876a67098131dd0b4d2f1691/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/Account.kt#L13-L16

or can we use just `publicKey`, I'm unsure of the use case for having `accountId` as well 🤔

Reply:
`accountId` is very useful because it's human-readable
in fact, in most places we use `accountId`, and `publicKey` is not used that much

Reply:
woops I mixed them up, I meant unsure of the `publicKey` use case. Do we need both? (ie do we need the ByteArray one?)

Reply:
I don't think we need the ByteArray. It was originally added there to match the SDK KeyPair, but I think we can omit it